python/global_scheduler/test/test.py

```diff
@@ -119,9 +115,12 @@ def tearDown(self):
     else:
       self.p1.kill()
     # Kill local schedulers, plasma managers, and plasma stores.
-    map(subprocess.Popen.kill, self.local_scheduler_pids)
-    map(subprocess.Popen.kill, self.plasma_manager_pids)
-    map(subprocess.Popen.kill, self.plasma_store_pids)
```

Comment:
this definitely works for me. Just confirmed it again on my mac at commit 3159a78ad795ea24920dc13a2ac8e2708d4bdf7f
`
00:08 $ ps -efww |egrep plasma
  501 65570 64833   0 12:09AM ttys008    0:00.00 egrep plasma
✔ ~/devrepo/github/ray-mydev [master|…32⚑ 8] 
00:09 $ ps -efww |egrep photon
  501 65606 64833   0 12:09AM ttys008    0:00.01 egrep photon
✔ ~/devrepo/github/ray-mydev [master|…32⚑ 8] 
00:09 $ ps -efww |egrep redis
  501 65642 64833   0 12:09AM ttys008    0:00.00 egrep redis
✔ ~/devrepo/github/ray-mydev [master|…32⚑ 8] 
`